ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -15,11 +16,40 @@ public ParkingSlotController(IParkingSlotService parkingSlotService)
             _parkingSlotService = parkingSlotService;
         }
 
-        public IActionResult Index(Guid parkingZoneId)
+        public IActionResult Index(Guid zoneId)
         {
-            var parkingSlots = _parkingSlotService.GetSlotsByZoneId(parkingZoneId);
-            var listItemVMs = ListItemVM.MapToVM(parkingSlots);
-            return View(listItemVMs);
+            var parkingSlots = _parkingSlotService.GetSlotsByZoneId(zoneId);
+            var listItemVMs = ListItemVM.MapToVM(parkingSlots).ToList();
+            var sorted = listItemVMs.OrderBy(item => item.Number);
```

Comment:
there is no need to make them sorted because we have such kind of task further (filtering)